tsnet/tsnet.go

```diff
@@ -323,6 +323,17 @@ func (s *Server) doInit() {
 	}
 }
 
+func (s *Server) TailscaleIPs() []netip.Addr {
```

Comment:
@crawshaw wanted to do something similar but I convinced him not to. If we really want this API to exist, it should take a context, have docs and return the error.

Reply:
Oh If we go the `s.lb.Netmap` route then we can skip the context and the error. Instead just document that it returns `nil` if the server isn't running or IPs are unavailable.

Reply:
Yeah. Seems good API to have and don't need context.

Reply:
Done.